qemu/tests/qemu_img.py

```diff
@@ -177,6 +178,42 @@ def create_test(cmd):
                 cluster_size=params.get("image_cluster_size"))
         remove(img)
 
+    def send_signal(timeout=360):
+        """
+        send signal "SIGUSR1" to qemu-img without the option -p
+        to report progress
+        """
+        logging.info("Send signal to qemu-img")
+        end_time = time.time() + timeout
+        while time.time() < end_time:
+            time.sleep(1)
+            status = utils.system("kill -SIGUSR1 `pidof qemu-img`",
+                                  ignore_status=True)
+            if status == 0:
+                return None
+        logging.info("Fail to get pid of qemu-img")
+
+    def check_command_output(CmdResult):
+        """
+        Check standard error or standard output of command
+        : param CmdResult: a list of CmdResult objects
+        """
+        logging.info("Check result of command")
+        check_output = params.get("check_output", "exit_status")
+        output = getattr(CmdResult, check_output)
+        if check_output == "exit_status":
+            if output == 0:
+                return None
+            else:
+                err_msg = "Get nonzero exit status '%s'" % CmdResult.command
```

Comment:
It's better to output the value of output.
"Get nonzero exit status(%d) '%s'" % (output, CmdResult.command)